code/game/jobs/role_authority.dm

```diff
@@ -336,11 +336,8 @@ sorts them out by their department.
 	if(late_join) //If they late joined, we put them in cryo.
 		M.loc = late_join
 	else
-		var/i
-		var/obj/effect/landmark/L //To iterate.
 		var/obj/effect/landmark/S //Starting mark.
-		for(i in landmarks_list)
-			L = i
+		for(var/obj/effect/landmark/L in landmarks_list) // This whole thing is fucking awful
```

Comment:
This should stay a typeless loop since the landmarks_list shouldn't contain anything else

Reply:
This is just standardisation since all other uses of landmarks_list are type checked the same way

Reply:
I don't care what the others are doing, this is incorrect